src/lib.rs

```diff
@@ -348,6 +363,36 @@ impl<'text> BidiInfo<'text> {
         }
     }
 
+    // /// Returns if the paragraph is Left direction, right direction or mixed.
```

Comment:
document
 - behavior for empty paras
 - behavior for paras that don't come from this object

Reply:
for empty para, done in the tests! shall we document?